### ISSUE TITLE
[MOD: Gunman Chronicles] Смена карты зависает на Node Graph out of Date. Rebuilding...

### ISSUE THREAD

**Issue by Vladislav4KZ** [FWGS/xash3d-fwgs#462]
Смена карты с rebar3d -> rebar3e зависает на "Node Graph out of Date. Rebuilding..."
Сохранение: 
[autosave.zip](https://github.com/FWGS/xash3d-fwgs/files/6575246/autosave.zip)

При загрузке этого же сохранения в GoldSource нет зависания, и даже это сообщение не появляется
Кстати, в папке graphs уже есть 2 файла rebar3e.nod и rebar3e.nrp созданные 28 мая (тогда я дошёл до этой карты и дальше не могу пройти)

**Comment by hiIexistlol** [comment]
такое случается часто и может быть потому, что уровни, как мне кажется, устарели.
(использовал гугл переводчик) 
(used google translate)

**Comment by Vladislav4KZ** [comment]
> это происходит часто и может быть потому, что уровни устарели, я думаю
> (использовал переводчик Google) 
> (used google translate)

Yes, I have seen this message several times during the game, but it is in this save after the changelevel that it hangs on this message.

The original Half-Life engine does not have this freeze in my savegame

**Comment by hiIexistlol** [comment]
is that save from goldsrc?
if not then hmm

**Comment by Vladislav4KZ** [comment]
> is that save from goldsrc?

This is save from xash, but it compatible with goldsource

And vice versa, saves from goldsource are compatible with xash

**Comment by Vladislav4KZ** [comment]
Такая же проблема есть и в Survive in Catacombs

**Comment by mittorn** [comment]
По идее если запускать без -dev, не должно зависать. Он зависает от сотен мегабайт сообщений отправляемых в лог

**Comment by Vladislav4KZ** [comment]
> По идее если запускать без -dev, не должно зависать. Он зависает от сотен мегабайт сообщений отправляемых в лог

Да кстати, выключил режим разработчика и перестало зависать

**Comment by mittorn** [comment]
Может ещё место под лог кончилось и от этого виснет, было где-то такое

**Comment by Vladislav4KZ** [comment]
> Может ещё место под лог кончилось и от этого виснет, было где-то такое

Точно нет, он не мог разом все 500гб занять
